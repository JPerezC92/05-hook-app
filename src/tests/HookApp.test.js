import { shallow } from "enzyme";
import HookApp from "../HookApp";

describe("Pruebas en <HookApp />", () => {
  test("Debe de compararse correctamente con la imagen", () => {
    const wrapper = shallow(<HookApp />);

    expect(wrapper).toMatchSnapshot();
  });
});
