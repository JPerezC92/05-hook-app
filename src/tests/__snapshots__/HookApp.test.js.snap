// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <HookApp /> Debe de compararse correctamente con la imagen 1`] = `
<div>
  <h1>
    HookApp
  </h1>
</div>
`;
